src/views/SystemProfile/components/ToolsAndSoftware/index.tsx

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Card,
+  CardBody,
+  CardGroup,
+  CardHeader,
+  Grid,
+  GridContainer,
+  Link
+} from '@trussworks/react-uswds';
+import classnames from 'classnames';
+
+import {
+  DescriptionDefinition,
+  DescriptionTerm
+} from 'components/shared/DescriptionGroup';
+import Divider from 'components/shared/Divider';
+import SectionWrapper from 'components/shared/SectionWrapper';
+import Tag from 'components/shared/Tag';
+import useCheckResponsiveScreen from 'hooks/checkMobile';
+import { GetCedarSystems_cedarSystems as CedarSystemProps } from 'queries/types/GetCedarSystems';
+
+import './index.scss';
+
+type ToolsAndSoftwareProps = {
+  className?: string;
+  children?: React.ReactNode;
+  system: CedarSystemProps;
+};
+
+type ProductDataTodo = {
+  id: string;
+  name: string;
+  manufacturer: string;
+  type: string;
+  tags?: string[];
+  version: string;
+  edition?: string;
+};
+
+const productsData: ProductDataTodo[] = [
+  {
+    id: '1',
+    name: 'Drupal',
+    manufacturer: 'Drupal Association',
+    type: 'Enterprise Content Management (ECM)',
+    version: '9.3',
+    edition: 'Enterprise'
+  },
+  {
+    id: '2',
+    name: 'Kong',
+    manufacturer: 'Kong Enterprise',
+    type: 'Software Development',
+    tags: ['API Gateway'],
+    version: '2.1'
+  }
+];
+
+const pointOfContactData = {
+  name: 'Greta May Jones',
+  role: 'Business Owner',
+  email: 'mailto:todo'
+};
+
+const SystemToolsAndSoftware = ({
+  className,
+  children,
```

Comment:
Are we expecting these components to render children?  It doesn't hurt to define this prop, but don't think at this time we planning to pass children to these subpages